#kiểm tra một khối để tìm lỗi
try: 
  print("Hello")
#Xử lý lỗi
except:
  print("Something went wrong")
#Thực thi mã khi không có lỗi
else:
  print("Nothing went wrong")
#Thực thi mã, bất kể kết quả
finally:
  print("The 'try except' is finished")